### ISSUE TITLE
cempakajaya.com: detection

### ISSUE THREAD

**Issue by king-iskan** [uBlockOrigin/uAssets#17005]
### Prerequisites

- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.)
- [X] I forced an update of my filter lists. (Click the ["Purge all caches"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#purge-all-caches) button while holding the 'Shift' key, then click the ["Update now"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#update-now) button.)
- [X] I did not remove any of the [default filter lists](https://user-images.githubusercontent.com/585534/91296476-ab9d8c00-e76a-11ea-9fb9-87d6861bdf73.png), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable external filter lists, or I have verified that the issue still occurs without enabling external filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin (uBO) along with other [similarly-purposed blockers](https://twitter.com/gorhill/status/1033706103782170625).
- [X] I have verified that the web browser's built-in blocker or [DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking) (standalone or through a VPN) is not causing the issue.
- [ ] I did not answer truthfully to **ALL** the above checkpoints.

### URL address of the web page

`https://cempakajaya.com/dashboard`

### Category

detection

### Description

detection after login

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.47.2
Chromium: 110
filterset (summary): 
  network: 85182
  cosmetic: 39246
  scriptlet: 17396
  html: 0
listset (total-discarded, last-updated): 
  default: 
    user-filters: 149-38, never
    easylist: 63278-47, 30m
    easyprivacy: 31000-1172, 30m
    plowe-0: 3750-928, 30m
    ublock-abuse: 79-0, 30m
    ublock-badware: 4612-123, 30m
    ublock-filters: 32963-107, 30m
    ublock-privacy: 286-1, 30m
    ublock-quick-fixes: 340-128, 30m
    ublock-unbreak: 1932-61, 30m
    urlhaus-1: 6216-0, 30m
filterset (user): [array of 59 redacted]
trustedset: 
  added: [array of 8 redacted]
  removed: 
    chrome-extension-scheme
modifiedUserSettings: [none]
modifiedHiddenSettings: [none]
supportStats: 
  allReadyAfter: 1166 ms (selfie)
  maxAssetCacheWait: 1972 ms
popupPanel: 
  blocked: 5
  blockedDetails: 
    adspeed.net: 1
    doubleclick.net: 1
    effacedefend.com: 2
    googleadservices.com: 1
```
</details>

**Comment by JobcenterTycoon** [comment]
Fixed with the other issue already https://github.com/uBlockOrigin/uAssets/commit/3905f84a3c4857ff4aec0f8c3c24af2cbb1b25f2

**Comment by king-iskan** [comment]
> ### Prerequisites
> * [x]  I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
> * [x]  I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.)
> * [x]  I forced an update of my filter lists. (Click the ["Purge all caches"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#purge-all-caches) button while holding the 'Shift' key, then click the ["Update now"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#update-now) button.)
> * [x]  I did not remove any of the [default filter lists](https://user-images.githubusercontent.com/585534/91296476-ab9d8c00-e76a-11ea-9fb9-87d6861bdf73.png), or I have verified that the issue was not caused by removing any of the default lists.
> * [x]  I did not enable external filter lists, or I have verified that the issue still occurs without enabling external filter lists.
> * [x]  I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
> * [x]  I am not using uBlock Origin (uBO) along with other [similarly-purposed blockers](https://twitter.com/gorhill/status/1033706103782170625).
> * [x]  I have verified that the web browser's built-in blocker or [DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking) (standalone or through a VPN) is not causing the issue.
> * [ ]  I did not answer truthfully to **ALL** the above checkpoints.
> 
> ### URL address of the web page
> `https://cempakajaya.com/dashboard`
> 
> ### Category
> detection
> 
> ### Description
> detection after login
> 
> ### Screenshot(s)
> Screenshot(s)
> ### Configuration

already close. cancel for this. i'm sorry.

**Comment by king-iskan** [comment]
> Fixed with the other issue already [3905f84](https://github.com/uBlockOrigin/uAssets/commit/3905f84a3c4857ff4aec0f8c3c24af2cbb1b25f2)

@JobcenterTycoon 
Need your help again, need to update. Because now detection again. I think dev already update. Thanks
